sdk/confidentialledger/azure-security-confidentialledger/src/samples/README.md

```diff
@@ -22,11 +22,35 @@ Getting started explained in detail [here](https://github.com/Azure/azure-sdk-fo
 ## Examples
 
 ```java readme-sample-createClient
+ConfidentialLedgerIdentityClientBuilder confidentialLedgerIdentityClientbuilder = new ConfidentialLedgerIdentityClientBuilder()
+    .identityServiceUri("https://identity.confidential-ledger.core.azure.com")
```

Comment:
This should be updated to `certificateEndpoint`. The code must be regenerated for certificate client too.